gio-2.0.pc.in

```diff
@@ -4,8 +4,8 @@ libdir=@libdir@
 includedir=@includedir@
 
 giomoduledir=@GIO_MODULE_DIR@
-glib_compile_schemas=@libdir@/glib-2.0/glib-compile-schemas
-glib_compile_resources=@libdir@/glib-2.0/glib-compile-resources
+glib_compile_schemas=glib-compile-schemas
+glib_compile_resources=glib-compile-resources
```

Comment:
I wonder if you can just avoid picking the previous commit from debian, instead of reverting it after. Or did you do it that way to keep history of the fact that we don't want to keep that commit from debian?